htmltest/check-img.go

```diff
@@ -64,6 +64,42 @@ func (hT *HTMLTest) checkImg(document *htmldoc.Document, node *html.Node) {
 		return
 	}
 
+	// Check usemap
+	if htmldoc.AttrPresent(node.Attr, "usemap") {
+		usemapRef := htmldoc.NewReference(document, node, attrs["usemap"])
+
+		if len(usemapRef.URL.Path) > 0 {
+			hT.issueStore.AddIssue(issues.Issue{
+				Level:     issues.LevelError,
+				Message:   "path in usemap attribute",
+				Reference: ref,
+			})
+		} else if len(usemapRef.URL.Fragment) == 0 {
+			hT.issueStore.AddIssue(issues.Issue{
+				Level:     issues.LevelError,
+				Message:   "usemap empty",
+				Reference: ref,
+			})
+		} else {
+			hT.checkInternalHash(usemapRef)
+		}
+
+		parent := node.Parent
+		if parent.Data == "a" {
+			hT.issueStore.AddIssue(issues.Issue{
+				Level:     issues.LevelError,
+				Message:   "img with usemap in a link",
```

Comment:
Perhaps tweak the wording to explain this isn't allowed

`<img> with usemap attribute not allowed as a descendant of am <a> tag`